Mathlib/CategoryTheory/Galois/Full.lean

```diff
@@ -0,0 +1,169 @@
+/-
+Copyright (c) 2024 Christian Merten. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Christian Merten
+-/
+import Mathlib.CategoryTheory.Galois.Examples
+import Mathlib.CategoryTheory.Galois.Prorepresentability
+
+/-!
+
+# Fiber functors are full
+
+Any (fiber) functor `F : C ⥤ FintypeCat` factors via the forgetful functor
+from finite `Aut F`-sets to finite sets. The induced functor
+`H : C ⥤ Action FintypeCat (MonCat.of (Aut F))` is shown to be full in this file.
```

Comment:
Are you not showing it is fully faithful?

Reply:
Well yes, but the faithfulness is easy and is essentially done in the `Basic` file. So this file is really about the `Full` part. Should I change the wording?

Reply:
Yes, I think you should say that the fiber functors are actually fully faithful by the top of the file.